gramex/apps/nlg/utils.py

```diff
@@ -151,17 +146,11 @@ def sanitize_fh_args(args, func=join_words):
 
 
 def check_grammar(text):
-    host = config.get('languagetool', 'hostname')
-    port = config.get('languagetool', 'port')
-    apiversion = config.get('languagetool', 'apiversion')
-    url = "{}:{}/{}/check?language=en-us&text={}"
-    try:
-        resp = requests.get(url.format(host, port, apiversion, text))
-        if resp.status_code == requests.codes.ok:
-            resp = resp.json()['matches']
-        else:
-            resp = []
-    except requests.ConnectionError:
+    url = 'https://localhost:9988/languagetool/grammar-check?q='
+    resp = requests.get(url + text)
```

Comment:
Please pass `{'q': text}` to requests to ensure proper URL escaping.

Secondly: avoid hardcoding port 9988. Instead, create `gramex.ml.languagetoolrequest` as a function that we can call directly to get the languagetool result. `gramex.ml.languagetool` internally should call this function.